src/NuGet.Services.EndToEnd/NuGet.Services.EndToEnd.csproj

```diff
@@ -152,6 +152,7 @@
     </None>
   </ItemGroup>
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" Version="1.11.4"/>
```

Comment:
Ideally we would update this dependency without lifting it (perhaps by updating the direct dependency that pulls it in). But if that's not feasible, this solution is fine.